app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js

```diff
@@ -803,6 +803,14 @@ define([
             },
             $.mage.__('Please enter a valid date.')
         ],
+        'validate-date-range': [
+            function (value, params) {
+                var fromDate = $('input[name*="' + params + '"]').val();
+
+                return (moment.utc(value).unix() >= moment.utc(fromDate).unix()) || isNaN(moment.utc(value).unix());
```

Comment:
Please remove the redundant parentheses in order to fix the static test failure.

Reply:
Hello @dmytro-ch 
I have update PR code as per your suggestion for remove the redundant parentheses to fix the static test. can you review it.